tests/installation/ipxe_install.pm

```diff
@@ -131,26 +131,43 @@ END_BOOTSCRIPT
     diag "$response->{status} $response->{reason}\n";
 }
 
+
 sub run {
     my $self = shift;
 
     poweroff_host;
 
-    #virtualization tests use a static ipxe configuration file
-    set_bootscript unless get_var('VIRT_AUTOTEST');
+    #virtualization tests use a static ipxe configuration file in O3
+    set_bootscript unless get_var('VIRT_AUTOTEST') && is_tumbleweed;
 
     set_pxe_boot;
 
     poweron_host;
 
+    select_console 'sol', await_console => 0;
+
+    #use ipxe bootloader in O3
+    #it is static menu and choose the TW entry to start installation
+    if (get_var('VIRT_AUTOTEST') && is_tumbleweed) {
```

Comment:
Why need get_var('VIRT_AUTOTEST') here ? Does virt makes any difference when comes to ipxe install ? Is this because virt installs baremetal host via ipmi ?

Reply:
the ipxe_install.pm is used in both OSD and O3 by kernel team, virt team and etc. The ipxe menu is only used in O3 by virt team. kernel team uses the ipxe install in a runtime way with ipmi backend, fg. post the install command based in different SUT.

Reply:
What about 
```
if (use static menu or similar)
```
? 
Because what is being done in the if is using static menu compared to customized bootscript.

Reply:
Static menu may vary in different places, thus the key or precedure to kick off a boot would be different. Also, introducing an extra setting or variable is not neccessary.